clients/cobol-lsp-vscode-extension/src/commands/ClearCopybookCacheCommand.ts

```diff
@@ -16,27 +16,27 @@ import * as path from "node:path";
 import * as vscode from "vscode";
 import { FileType } from "vscode";
 import {
-  C4Z_FOLDER,
   CLEARING_COPYBOOK_CACHE,
   COPYBOOK_CACHE_CLEARED_INFO,
   COPYBOOKS_FOLDER,
+  ZOWE_FOLDER,
 } from "../constants";
+import { Utils } from "../services/util/Utils";
 
 /**
- * Clears the downloaded copybook cache folder ({workspace}/.c4z/.copybooks).
+ * Clears the downloaded copybook cache folder ({Utils.getExtensionFolder()}/.zowe/.copybooks).
  *
  */
 export function clearCache() {
   vscode.window.setStatusBarMessage(
     CLEARING_COPYBOOK_CACHE,
     Promise.resolve().then(
       () => {
-        const firstWorkspaceFolder = vscode.workspace.workspaceFolders![0];
-        const folderUri = firstWorkspaceFolder!.uri;
-        const fileUri = folderUri.with({
-          path: path.join(folderUri.fsPath, C4Z_FOLDER, COPYBOOKS_FOLDER),
+        const folderUri = vscode.Uri.file(Utils.getExtensionFolder());
+        const zoweFileUri = folderUri.with({
+          path: path.join(folderUri.fsPath, ZOWE_FOLDER, COPYBOOKS_FOLDER),
         });
-        deleteFolderContent(fileUri);
+        deleteFolderContent(zoweFileUri);
         vscode.window.showInformationMessage(COPYBOOK_CACHE_CLEARED_INFO);
       },
       () =>
```

Comment:
Check with the maintainers if they have a convention of keeping the messages in one place and use variables to refer them.

Reply:
not sure I follow, the line isn't `vscode.window.showInformationMessage(COPYBOOK_CACHE_CLEARED_INFO);`
changed in this PR

Reply:
This was about the message below, I thought it was added in this pr...